common/services/move.test.js

```diff
@@ -892,6 +892,47 @@ describe('Move Service', function() {
     })
   })
 
+  describe('#unassign()', function() {
+    context('without move ID', function() {
+      it('should reject with error', function() {
```

Comment:
Does this pass? The method doesn't seem to include this like the other methods do.

Would be good to add it though and ensure these tests are functioning as expected.

Reply:
It passes. It's just called the its own `update` method rather than the api-client's. I don't understand what you mean by this or by adding it?

Reply:
Ah yes, sorry I missed that. That's probably a nicer way to handle it actually.